### ISSUE TITLE
express-ipfilter depends on vulnerable versions of ip

### ISSUE THREAD

**Issue by amatiash** [jetersen/express-ipfilter#185]
<img width="705" alt="Screenshot 2024-09-20 at 00 25 15" src="https://github.com/user-attachments/assets/06d7d92c-a9b3-44c8-a504-2c05aba79123">

**Comment by JackHammer29** [comment]
This was fixed in recent merge

**Comment by sceccotti89** [comment]
@JackHammer29 I don't think it is. I'm still getting the same vulnerability running a simple npm audit.
I'm using version 1.3.2
